Source/Android/app/src/main/res/values/strings.xml

```diff
@@ -579,17 +579,7 @@
     <string name="convert_warning_nkit">Dolphin can\'t convert NKit files to non-NKit files. Converting an NKit file in Dolphin will result in another NKit file.\n\nIf you want to convert an NKit file to a non-NKit file, you can use the same program as you originally used when converting the file to the NKit format.\n\nDo you want to continue anyway?</string>
     <string name="convert_success_message">The disc image was successfully converted.</string>
     <string name="convert_failure_message">Dolphin failed to complete the requested action.</string>
-    <string name="convert_format_info">
-ISO: A simple and robust format which is supported by many programs. It takes up more space
-than any other format.
-\n\nGCZ: A basic compressed format which is compatible with most versions of Dolphin and some
-other programs. It can\'t efficiently compress junk data (unless removed) or encrypted Wii data.
-\n\nWIA: An advanced compressed format which is compatible with Dolphin 5.0-12188 and later,
-and a few other programs. It can efficiently compress encrypted Wii data, but not junk data
-(unless removed).
-\n\nRVZ: An advanced compressed format which is compatible with Dolphin 5.0-12188 and later.
-It can efficiently compress both junk data and encrypted Wii data.
-    </string>
+    <string name="convert_format_info">ISO: A simple and robust format which is supported by many programs. It takes up more space than any other format.\n\nGCZ: A basic compressed format which is compatible with most versions of Dolphin and some other programs. It can\'t efficiently compress junk data (unless removed) or encrypted Wii data.\n\nWIA: An advanced compressed format which is compatible with Dolphin 5.0-12188 and later, and a few other programs. It can efficiently compress encrypted Wii data, but not junk data (unless removed).\n\nRVZ: An advanced compressed format which is compatible with Dolphin 5.0-12188 and later. It can efficiently compress both junk data and encrypted Wii data.</string>
```

Comment:
The script needs to parse the strings in the same way as Android does. If we apply band-aids like this instead of fixing the problem at its root, sooner or later someone is going to add a new string and forget to apply this band-aid and inadvertently break translations.

Reply:
Hm, I guess I could remove `\n` and spaces at the start/end of a string then?

Reply:
What I'm guessing is that Android folds all runs of whitespace (including newlines) into a single space. If you can confirm that that's what Android does, I think that's what we should do.

Reply:
Ok I figured out that Android does parse the line breaks in strings.xml as spaces. This just sucks for whenever we want to combine with translations. I *could* try and remove those trailing spaces but is that even worth it?